### DISCUSSION TITLE
Welcome to misskey Discussions!

### DISCUSSION THREAD

**Discussion by syuilo** [misskey-dev/misskey#6947] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by TheKroMang** [comment]
I think there should be buttons to add the effects (like bounce and spin) to posts without having to type it out. highlighting text before clicking that button should contain the selected text in the effect. 😇
